runtime/interpreter/interpreter.go

```diff
@@ -3239,7 +3239,9 @@ var converterFunctionValues = func() []converterFunction {
 			if converterFunctionValue.NestedVariables == nil {
 				converterFunctionValue.NestedVariables = map[string]*Variable{}
 			}
-			converterFunctionValue.NestedVariables[name] = NewVariableWithValue(value)
+			// these variables are do need to be metered as they are only ever declared once,
+			// and can be considered base interpreter overhead
+			converterFunctionValue.NestedVariables[name] = NewVariableWithValue(nil, value)
```

Comment:
I guess the comment should be 'not metered'
```suggestion
			// these variables are not needed to be metered as they are only ever declared once,
			// and can be considered base interpreter overhead
			converterFunctionValue.NestedVariables[name] = NewVariableWithValue(nil, value)
```

Maybe add a new 'NewUnmeteredVariableWithValue' constructor, just so that its more explicit